### PR TITLE
[ADD] sms marketing: added essentials/getting started docs

### PR DESCRIPTION
**Pull request by ksc-odoo**
Task ID: [2942125](https://www.odoo.com/web#id=2942125&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hi @mivu-odoo 

I added some basic, "essential," documentation to the SMS Marketing doc.

Once I get your approval/review, I'll pass it along to Zac.

Thanks!

**Comment by mivu-odoo** [comment]
Hi @ksc-odoo!

This PR wasn't passing the `ci/documentation` code check b/c of some build errors.

As of commit [d542657](https://github.com/odoo/documentation/commit/d5426579f41ab6bfac8a8cacdc7c74da670c42e5)

- Renamed the RST files to replace the hyphens with underscores in `contacts-blacklist.rst` and `sms-integrations.rst`
- Renamed `first-campaign.rst` to `first_sms_campaign.rst` to be more specific b/c I know PR #2552 is going to add a file named `first_campaign.rst` for Marketing Automation user docs, and I thought it would be confusing to have two files in different places with the same name
- Fixed the RST file names referenced in `essentials.rst` to match the new RST file names (the ones with the underscores)
- Renamed the image folders to replace the hyphens with underscores (so they match the new RST file names)
- Fixed the image folder name references in `first_sms_campaign.rst`, `sms_integrations.rst`, and `contacts_blacklist.rst`
- Fixed the indentation of `sms_marketing/essentials` and `sms_marketing/pricing` in `sms_marketing.rst` to align with the first colon of `:titlesonly:`

Despite all these changes, the PR still won't pass the code check b/c there's one error I haven't quite figured out how to crack. 

When I run `make html`, I get an error for `first_sms_campaign.rst:140: WARNING: unknown document: pricing_and_faq`. I tried several different iterations like including more of the file path, but I would still get the same build error. Will probably have to loop in ZST to fix.

**Comment by samueljlieber** [comment]
Hi @mivu-odoo and @ksc-odoo 👋

I fixed what was causing the build error here! It was due to an incorrect referenced doc path on line 140 of `first_sms_campaign.rst` 🙂

The path was: 
`pricing_and_faq`

The referenced doc path needs to be [relative to the document](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#contributing-relative-links) you are linking from, so the correct path looks like this:
`../pricing/pricing_and_faq`

**Comment by ksc-odoo** [comment]
@StraubCreative Ready for your review!

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

These docs need another round of proofreading and revisions.

Lots of 2nd-person _you/your_ language, some screenshots are too big, unnecessary, or not named properly, etc...the usual suspects 😉 

Please submit updated content to @samueljlieber he'll handle the git flow for you.

**Comment by samueljlieber** [comment]
Hi @ksc-odoo, in b4acc53 I implemented your changes per @StraubCreative [suggestions](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1284380372):
- FYI renamed the following files:
    - `first_sms_campaign.rst` to `sms_essentials.rst`
    - `sms_integrations.rst` to `sms_campaigns.rst`
    - `contacts_blacklists.rst` to `mailing_lists_blacklists.rst`
- `sms_essentials.rst` lines `29`-`35` & lines `136`-`142` I formatted as a list rather than sentences because they are referring to options that can be selected from a dropdown, it made sense to me for the options to be displayed in a list.

I think your changes to these docs look really good 👍 I made a few small RST adjustments, @ksc-odoo can you please confirm these changes with a review before I pass to @StraubCreative? 🙏

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo, in [b4acc53](https://github.com/odoo/documentation/commit/b4acc5309e3bb6ad12dffbd3c88b9923a2ee9225) I implemented your changes per @StraubCreative [suggestions](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1284380372):
> 
> * FYI renamed the following files:
>   
>   * `first_sms_campaign.rst` to `sms_essentials.rst`
>   * `sms_integrations.rst` to `sms_campaigns.rst`
>   * `contacts_blacklists.rst` to `mailing_lists_blacklists.rst`
> * `sms_essentials.rst` lines `29`-`35` & lines `136`-`142` I formatted as a list rather than sentences because they are referring to options that can be selected from a dropdown, it made sense to me for the options to be displayed in a list.
> 
> I think your changes to these docs look really good 👍 I made a few small RST adjustments, @ksc-odoo can you please confirm these changes with a review before I pass to @StraubCreative? 🙏

Hey @samueljlieber - Feel free to pass it along to @StraubCreative 👍

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:107]: Note for all of the headings that start with `How to...`
We can just remove that part and stick with the action that the heading conveys for the section. Please apply to all headings in this and other docs.

```suggestion
Track links used in SMS messages
--------------------------------
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:127]: Too wide. Doesn't appear to be in HD.

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:137]: Point of clarity: what does `excessive` mean here? How is this defined?

Consider the following suggestion, instead:
```suggestion
- :guilabel:`Send`: sends the message immediately. Consider using this option if the recipient list
  is highly refined or in cases that involve fast approaching deadlines, such as a flash sale.
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:149]: I think in this case, even though it's solitary, the second tag is actually a `menuselection`. As well, I would indicate`filters` and `measures` as `guilabels`. Can remove the parenthesis in the last sentence too and just write out what those views are.
```suggestion
On the :guilabel:`Reporting` page (accessible via the :menuselection:`Reporting` option in the
header menu), there are options to apply different combinations of :guilabel:`Filters` and
:guilabel:`Measures` to view metrics in a number of different layouts, as in the :guilabel:`Graph`,
:guilabel:`List`, and :guilabel:`Cohort` views.
```

File: content/applications/marketing/sms_marketing/essentials/sms_essentials.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/sms_essentials.rst:158]: The crop is pretty good here, however this image is too wide @ 1920 pixels. Consider taking a shot that has more data, as well, instead of what we get by default using runbot. Images that show the depth of capabilities, I think, are more valuable to the reader and justifies adding the extra file weight to the page.

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

Nice work. I like the depth of explanation and walk-through on these 👍 

My CRs below are mostly around small things and fine tuning since the content already hits the mark. See below where I made comments just on the `sms_essentials.rst` doc,   which apply to all docs in this PR, as well. 

Overview:
- formatting of sentences, mainly parenthesis use. I see what you're doing in most cases, using parenthesis as tiny _aside_ comments in sentences, however, I don't think they're needed in most cases. Most of what I see comes off as redundant or could easily be included into the sentence with a comma.
- line breaks. Just a few spots to fix.
- headings. Two headings are duplicates and we don't need `How to...` prefixes.
- images. Most images are too large, not in HD, and could use red markup to illustrate cause > effect. As well, some screenshots would benefit from showcasing more information or a more complete use case to illustrate the depth of the product.

Can you take a look and apply to every doc on the PR? Can work with @samueljlieber to push up the next batch of changes.

Thank you both!

**Comment by samueljlieber** [comment]
Hi @ksc-odoo I implemented your changes in https://github.com/odoo/documentation/commit/ac8b1508a281defa2f2bfef8e010e14fab573d61 🙂

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Rewording to remove passive/conditional tense in favor of active.
```suggestion
Access contacts information from one or more mailing lists navigate to :menuselection:`Mailing Lists
--> Mailing List Contacts` to reveal a dashboard with with all the contacts associated with one or
more of the configured mailing lists in the database.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Don't think these get guilabels since it's not active instruction on the UI.
```suggestion
An accessible record of every sent mailing(s) is kept on each recipient's *chatter* section, located
beneath a recipient's contact form (in the *Contacts* application).
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Same here.
When referencing BE docs they don't use guilabels or capitalize this term.
We can mirror that style, for now, and can clarify at some point.
```suggestion
Database users can reference the chatter to easily keep track of communications, and see a history
of interactions with contacts and prospects.

For example, sales representatives can use the chatter to quickly find out which :abbr:`SMS (Short
Message Service)` mailing promotions a certain customer has received (or not received).
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: No guilabel here
```suggestion
Odoo *SMS Marketing* has a :guilabel:`Blacklist` feature that provides recipients with the power to
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Formatting
```suggestion
This is also known as the unsubscribe process: customers will automatically be added onto a
*blacklist*, if they click :guilabel:`Unsubscribe`, via their :guilabel:`Subscription Management`
page. Customers can also be added manually to the blacklist, if necessary.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: Wording/formatting.
```suggestion
To see a complete collection of blacklisted numbers, navigate to the :menuselection:`SMS Marketing
app --> Configuration --> Blacklisted Phone Numbers` to reveal a dashboard containing every
blacklisted phone number in the database.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: "Odoo reveals a separate page" phrase is too wordy. Just say what's happening in clear terms 😉 
```suggestion
To manually add a number to a blacklist, click the :guilabel:`Create` button in the upper-left
corner of the dashboard and enter the phone number on the next page's form. There's also a
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]: `Blacklisted Phone Numbers` gets a guilabel because it's a menu item and the name of a dashboard, while _blacklist_ does not. Added some more specificity here too.

```suggestion
Once the form is completed, click :guilabel:`Save` to add it to the :guilabel:`Blacklisted Phone
Numbers` list. To remove any number from the blacklist, select the desired number on the dashboard,
and then, on the phone number's form, click :guilabel:`Unblacklist`.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]:
```suggestion
contacts. This would include customers, who have already asked to be blacklisted` on :abbr:`SMS
(Short Message Service)` mailings.
```

File: content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst
Comment (StraubCreative) [inline content/applications/marketing/sms_marketing/essentials/mailing_lists_blacklists.rst]:
```suggestion
To do that, navigate to :menuselection:`SMS Marketing app --> Configuration --> Blacklisted Phone
Numbers`, and then select the :guilabel:`Favorites` drop-down menu (beneath the search bar), and
click :guilabel:`Import records`.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 👋 

A few more change requests here. See below and on [01d323d](https://github.com/odoo/documentation/pull/2553/commits/01d323d8b8a0883aeee72e3c1bb164683bb79d9d). 

If you agree with the changes, let me know so I can squash/merge, otherwise indicate if and where you might want to go over something in more detail.

Ty/lmk 🤙

**Review by StraubCreative — CHANGES_REQUESTED** [review]
EDIT: see [review notes above](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1358117349). GH bugged out and submitted the review twice 😓

**Comment by ksc-odoo** [comment]
> EDIT: see [review notes above](https://github.com/odoo/documentation/pull/2553#pullrequestreview-1358117349). GH bugged out and submitted the review twice 😓



> If you agree with the changes, let me know so I can squash/merge, otherwise indicate if and where you might want to go over something in more detail.

Hey @StraubCreative I agree with ALL these suggested changes - feel free to squash/merge at your earliest convenience. Thanks!

**Comment by StraubCreative** [comment]
@robodoo r+

**Review by StraubCreative — APPROVED** [review]

**Pull request closed without merging**
